### DISCUSSION TITLE
How to import/export signals and queries solely?

### DISCUSSION THREAD

**Discussion by tobias-graf-p** [datalust/seq-tickets#2057] [Help] [answered]
I have multiple Seq instances for PROD/STAGING/DEV environments. Is there a way how I can export signals and queries from one instance and import them in another one? Or any way to sync these (and other) settings between instances?

**Comment by liammclennan** [comment, marked as answer]
Hi @tobias-graf-p 

You can do this with `seqcli template export` and `seqcli template import`

https://docs.datalust.co/docs/command-line-client#template-export
